libnmstate/schema.py

```diff
@@ -247,3 +249,18 @@ class OVSBridge(object):
     PORT_NAME = 'name'
     PORT_VLAN_MODE = 'vlan-mode'
     PORT_ACCESS_TAG = 'access-tag'
+
+
+class Team(object):
+    TYPE = InterfaceType.TEAM
+    CONFIG_SUBTREE = InterfaceType.TEAM
+
+    PORTS_SUBTREE = 'ports'
```

Comment:
OVSBridge uses singular `port` instead of `ports`, I guess we cannot change the value since it comes from the schema but since the constant names can be different, maybe use `PORT_SUBTREE` and `class Port` to keep it uniform for Nmstate. What do you think?

Reply:
That seems fine. Thanks!